tiledb/sm/query/reader.cc

```diff
@@ -1537,16 +1537,27 @@ Status Reader::compute_result_coords(
   for (auto& result_tile : *result_tiles)
     tmp_result_tiles.push_back(&result_tile);
 
-  // Read and unfilter coordinate tiles
-  // NOTE: these will ignore tiles of fragments with format version >=5
+  // Preload zipped coordinate tiles. Note that this will
+  // ignore fragments with a version >= 5.
+  RETURN_CANCEL_OR_ERROR(load_offsets({constants::coords}));
+
+  // Preload unzipped coordinate tiles. Note that this will
```

Comment:
This preloads unzipped coordinate tile "offsets" (missing "offsets" word confused be slightly).

Reply:
Definitely, good catch. I've updated the comment here and for the other similar comment on L1540.